### ISSUE TITLE
Schema validity and id attributes

### ISSUE THREAD

**Issue by jonathanrobie** [openscriptures/morphhb#84]
When I validate against the [OSIS schema](https://crosswire.org/osis/osisCore.2.1.1.xsd) using xmllint, I get validation errors for the id attributes on words:
 
```
1Kgs.xml:321: element w: Schemas validity error : Element '{http://www.bibletechnologies.net/2003/OSIS/namespace}w', attribute 'id': The attribute 'id' is not allowed.
1Kgs.xml:322: element w: Schemas validity error : Element '{http://www.bibletechnologies.net/2003/OSIS/namespace}w', attribute 'id': The attribute 'id' is not allowed.
!!! SNIP !!!
```

Am I missing something?

**Comment by DavidTroidl** [comment]
I have an email thread going on this issue, trying to resolve the validation problem. The @ID in OSIS is capitalized, but the main issue is that XML IDs are not allowed to start with a number. I did a temporary correction for this when I validated 1Sam to check the two token issue.

**Comment by jonathanrobie** [comment]
So perhaps a global search/replace that replaces `id="` with `ID="id` would do the trick?

**Comment by DavidTroidl** [comment]
Not exactly. I replace: id=" with: ID="a, for a temporary check. I am still conversing with the authors of the IDs to see how they can be adjusted.

**Comment by DavidTroidl** [comment]
Sorry, I missed the point. Yes, yours would work.
